backend/swagger.yaml

```diff
@@ -20,3 +22,175 @@ paths:
+          schema:
+            $ref: '#/responses/badRequest'
+    put:
+      summary: Update todo item
+      operationId: update
+      tags:
+        - Item
+      parameters:
+        - name: body
+          in: body
+          schema:
+            $ref: '#/definitions/itemInput'
+      responses:
+        '200':
+          description: Updated todo item
+          schema:
+            $ref: '#/definitions/item'
+        '400':
+          description: Invalid input
+          schema:
+            $ref: '#/responses/badRequest'
+
+  /item/{id}:
+    get:
+      summary: Get todo item
+      description: Gets todo item by ID provided
+      operationId: get
+      tags:
+        - Item
+      parameters:
+        - type : integer
+          format: int64
+          name: id
+          in: path
+          required: true
+      responses:
+        '200':
+          description: 'Found todo item by ID'
+          schema:
+            $ref: '#/definitions/item'
+        '400':
+          description: 'Invalid ID'
+          schema:
+            $ref: '#/responses/notFound'
+        '404':
+          description: 'Not Found'
+          schema:
+            $ref: '#/responses/notFound'
+    delete:
+      summary: Delete todo item
+      description: Deletes todo item by ID
+      operationId: delete
+      tags:
+        - Item
+      parameters:
+        - type: integer
+          format: int64
+          name: id
+          in: path
+          required: true
+      responses:
+        '204':
+          $ref: '#/responses/deleted'
+        '400':
+          $ref: '#/responses/notFound'
+
+
+responses:
+  'deleted':
+    description: Sucessful
+    schema:
+      type: string
+      description: Successfully deleted
+  'badRequest':
+    description: Bad Request
+    schema:
+      $ref: '#/definitions/errorResponse'
+  'notFound':
+    description: Not Found
+    schema:
+      description: Primary resource not found
+
+definitions:
+  errorResponse:
+    title: Error Response
+    type: object
+    x-go-type:
+      import:
+        package: 'github.com/SAKA-club/todo/backend/errs'
+        alias: 'errs'
+      type: 'TodoError'
+    properties:
+      code:
+        type: string
+      message:
+        type: string
+      field:
+        type: string
+
+  item:
+    type: object
+    required:
+      - id
+      - title
+    properties:
+      id:
+        type: integer
+        format: int64
+      title:
+        type: string
+      body:
+        type: string
+      priority:
+        type: boolean
+      finish_by:
+        type: string
+        format: date
+        example: "2022-05-05"
+
+  itemInput:
+    type: object
+    required:
+      - title
+    properties:
+      id:
+        type: integer
+        format: int64
+      title:
+        type: string
+      body:
+        type: string
+      priority:
+        type: boolean
+      finish_by:
+        type: string
+        format: date
+        example: "2022-05-05"
```

Comment:
These changes seem to correlate better with #10 . Let's hold off on adding these in until we pickup that ticket.